tests/phpunit/tests/test-canonical.php

```diff
@@ -118,6 +109,8 @@ public function init_for_sitemaps() {
 
 	public static function wpTearDownAfterClass() {
 		_unregister_post_type( 'pllcanonical' );
+		_unregister_post_type( 'trcpt' );
+		_unregister_post_type( 'cpt' );
```

Comment:
Do we need to unregistered taxonomies as well, because I've added some in the fixtures as well?

Reply:
Good idea.

Reply:
Although, it might already be the case, see the doc comment for this function: https://github.com/WordPress/wordpress-develop/blob/97b2f07d2eba6cdd88bb960ba539d4c07f84f71f/tests/phpunit/includes/utils.php#L377-L382